src/Components/Api.jsx

```diff
@@ -4,25 +4,25 @@ const gamesApi = axios.create({
   baseURL: "https://games-nc.herokuapp.com/api/",
 });
 
-export const FetchAllReviews = () => {
+export const fetchAllReviews = () => {
   return gamesApi.get("reviews").then((res) => {
     return res.data;
   });
 };
 
-export const FetchAllCategories = () => {
+export const fetchAllCategories = () => {
   return gamesApi.get(`categories`).then((res) => {
     return res.data;
   });
 };
 
-export const FetchAllCategoryReviews = (category) => {
+export const fetchAllCategoryReviews = (category) => {
   return gamesApi.get(`reviews?category=${category}`).then((res) => {
     return res.data;
```

Comment:
if you wanted to make this DRYer (which I would recommend doing at some point as we'll want to use other queries on our get reviews endpoint such as sort by/order), look up axios request options/params object, using this means that axios will handle queries for us! No need for a separate function to get reviews by category